src/WebApiContrib.Core/MvcOptionsExtensions.cs

```diff
@@ -24,5 +29,44 @@ public static void UseGlobalRoutePrefix(this MvcOptions opts, IRouteTemplateProv
         {
             opts.Conventions.Insert(0, new FromBodyApplicationModelConvention(controllerPredicate, actionPredicate, parameterPredicate));
         }
+
+        /// <summary>
+        /// Registers a Validation action filter as application model convention, which applies this filter to POST, PUT, and DELETE actions.
+        /// </summary>
+        /// <param name="controllerPredicate">A controller predicate for further configuration.</param>
```

Comment:
`A controller predicate for further configuration.` please change to `A controller predicate for more granular application of the filter.` (please do the same on other methods and predicate comments)